addons/project/migrations/10.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+# Delete XML IDs
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
```

Comment:
Do not put this comment here. Actually not anywhere as it is very clear from the code what is done.